### DISCUSSION TITLE
How to send modifier + Tab properly

### DISCUSSION THREAD

**Discussion by kjuq** [blinksh/blink#1887] [Q&A] [answered]
# Answer

Add custom presses in `Blink > Settings > Keyboard > Custom Presses > Add > ACTION > SEND Hex Code`.

Ctrl-Tab: 1B5B32373B353B397E
Ctrl-Shift-Tab: 1B5B32373B363B397E
Alt-Tab: 1B09

# FYI

I couldn't find a way to use Alt-Shift-Tab yet. Please tell me how to achieve it if you know it.

**Comment by iflowerml** [comment, marked as answer]
Hello! We've recently added a guide for implementing `Ctrl-Tab` and `Ctrl-Shift-Tab`. 
For anyone that still needs help. Check it out [here!](https://github.com/blinksh/blink/discussions/1941)
